locales/fr/messages.properties

```diff
@@ -0,0 +1,223 @@
+##########################################
```

Comment:
Not sure what that means - should I delete it in my branch and re commit the PR?